src/main/webapp/resources/js/pages/projects/metadata/MetadataTemplatesList.jsx

```diff
@@ -25,16 +43,62 @@ import { unwrapResult } from "@reduxjs/toolkit";
  * @constructor
  */
 export function MetadataTemplatesList({ projectId }) {
-  const { canManage } = useSelector((state) => state.project);
   const { templates, loading } = useSelector((state) => state.templates);
+  const { canManage } = useSelector((state) => state.project);
+
+  const { fields } = useSelector((state) => state.fields);
+
   const dispatch = useDispatch();
   const [BASE_URL] = React.useState(() =>
     setBaseUrl(`/projects/${projectId}/metadata-templates`)
   );
 
+  const [templatesModified, setTemplatesModified] = React.useState([{}]);
+
   /**
-   * This crates the "actions" that appear at the right of every row in
-   * the table: field count, download template, and remove template (if applicable).
+   * Set default metadata template for project.
+   *
+   * @param {number} templateId - identifier for the metadata template to set as default
+   */
+  const setDefaultTemplate = async (templateId) => {
+    await dispatch(setDefaultTemplateForProject({ projectId, templateId }))
```

Comment:
Don't need await here since your using chaining

Reply:
Removed in 3879ffab4fe9c06d7f50602f38d2722fe07ced1a